workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -24,7 +24,8 @@ public class RequiredFieldInjector
 
         private const string LoggerName = nameof(RequiredFieldInjector);
         private const string BadRequiredMemberWarning
-            = "[Require] attribute found on member that is not Injectable. This member will be ignored.";
+            = "[Require] attribute found on member that is not Injectable. This member will be ignored. " 
+            + "Please make sure that types marked with [Require] are located in the <component name>.Requirable or WorldsCommands.Requirable namespace and are not part of the ECS API of the SpatialOS GDK.";
```

Comment:
I think " and are not part of the ECS API of the SpatialOS GDK." is impossible when combined with the previous part of the sentence

Reply:
I mean, is it possible for a part of `.Requirable` to be non-requirable?

Reply:
Removed the second part.